src/main/webapp/i18n/en/conversation.json

```diff
@@ -236,6 +237,12 @@
                         "true": "Announcement Channel",
                         "false": "Unrestricted Channel"
                     },
+                    "isCourseWideChannelInput": {
+                        "label": "Channel Scope",
+                        "explanation": "In a course-wide channel, all users in the course are automatically added. In a targeted channel, you can manually select the users to be added after creation.",
+                        "true": "Course-wide Channel",
+                        "false": "Targeted Channel"
```

Comment:
I don't really like targeted channel. Can you propose a different name?

Reply:
I changed 'Targeted' to 'Selective'